editor/assets/stylesheets/_z-index.scss

```diff
@@ -13,8 +13,8 @@ $z-layers: (
 	'.editor-post-visibility__dialog': 30,
 	'.editor-post-schedule__dialog': 30,
 	'.editor-block-mover': 30,
-	'.components-notice-list': 100000,
-	'.components-popover': 100000,
```

Comment:
I would call this a modal. The gray background should prohibit expanding the admin menus in this case.